### ISSUE TITLE
Delayed editor creation on load

### ISSUE THREAD

**Issue by fredck** [CKEditor5/ckeditor5.github.io#27]
Currently, it is pretty noticeable the delay on creating the editor in the website. You can even clearly see the ugly editor contents before it is replaced:

![flickering](https://cloud.githubusercontent.com/assets/630060/23895559/69c0d50e-08a6-11e7-8d8b-c6bfdcc836ec.gif)

**Comment by Reinmar** [comment]
I've talked with @oleq about this recently. I think that that FOUC cannot be easily avoided by pulling the styles out of the JS file and moving them to the `<head>` because these styles base on classes which are added later. So either we add separate styles (independent from the editor, targeted at styling the content when the editor is not available) to that website's stylesheet or we just hide the content before editor loads. I don't like the latter – the former seems more correct to me.

**Comment by fredck** [comment]
Small devil question... considering that this site is a demo of CKEditor, which comes in a script... does it make sense to have the script at the end of the page instead of "head". There is nothing else in the page for the user to contemplate, anyway ;)

**Comment by Reinmar** [comment]
AFAIK this will also block rendering the rest of the website and loading the other files – IMO, it's better to show something than to show nothing for the entire time.

**Comment by fredck** [comment]
Exactly... but that "something" is the problem... it's more like "whatever" and I feel that "nothing" is better, then.

Maybe we put the script element right before the editor contents then?

Or maybe move the Hotjar and Google Tag Manager scripts to the button as well and see what happens?

**Comment by oleq** [comment]
I would simply add the styles to `<div id="editor">` to give it the same width and background as the editable of the actual editor. So when the editor loads, to the user, it would look like `<div id="editor">` is being "boxed", transformed into the editor, which is acceptable.
